### PR TITLE
zk log change

### PR DESCRIPTION
**Pull request by pantianying**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: registry/zookeeper/registry.go [hunk 1]
```diff
@@ -381,7 +381,7 @@ func (r *zkRegistry) registerTempZookeeperNode(root string, node string) error {
 	}
 	zkPath, err = r.client.RegisterTemp(root, node)
 	if err != nil {
-		logger.Errorf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
+		logger.Warnf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
 		return perrors.WithMessagef(err, "RegisterTempNode(root{%s}, node{%s})", root, node)
```
Comment (hxmhlt) [inline registry/zookeeper/registry.go]: I think not all error's log level need set to warn. You can specify the 'node already exists' error type and set its log level to warn.

Comment (pantianying) [reply registry/zookeeper/registry.go]: i will fix it

File: registry/zookeeper/registry.go [hunk 2]
```diff
@@ -381,7 +381,11 @@ func (r *zkRegistry) registerTempZookeeperNode(root string, node string) error {
 	}
 	zkPath, err = r.client.RegisterTemp(root, node)
 	if err != nil {
-		logger.Errorf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
+		if err == zk.ErrNodeExists {
+			logger.Warnf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
+		} else {
+			logger.Errorf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
+		}
 		return perrors.WithMessagef(err, "RegisterTempNode(root{%s}, node{%s})", root, node)
```
Comment (hxmhlt) [inline registry/zookeeper/registry.go]: Is condition (err==zk.ErrNodeExists )should return perrors.WithMessagef?

Comment (pantianying) [reply registry/zookeeper/registry.go]: > This function will return error. And outer function will catch the error ,continue to call logger.Error. If you need to ignore the error, I think you should modify all error catch.

For now, there is no way to print this error except by calling the Interface Methods peripherally.

**Pull request merged**